Resources/js/router.js

```diff
@@ -46,7 +46,7 @@ class Router {
      * @param {Object} data
      */
     setRoutingData(data) {
-        this.setBaseUrl(data['base_url']);
+        this.setBaseUrl(data['base_url'] && data['base_url'].length > 0 ? data['base_url'] : window.location.host);
```

Comment:
```suggestion
        this.setBaseUrl((data['base_url'] && data['base_url'].length > 0) ? data['base_url'] : window.location.host);
```

Reply:
I have applied your recommendation, but the build process remove them. So build rep does not change.